src/common/components/datepicker/TimesList.tsx

```diff
@@ -111,7 +111,7 @@ const TimeItem: React.FC<TimeItemProps> = ({
   // scroll focused element into view
   React.useEffect(() => {
     if (focused) {
-      buttonRef.current?.scrollIntoView({
+      buttonRef.current?.scrollIntoView?.({
```

Comment:
I'm only curious here, but what is the reason for ?-mark in scrollIntoView?

Reply:
jsdom seems to not support that one and errors out. Maybe need add comment about that?

Reply:
That's something that I was expecting. Maybe comment could be helpful. It was not obvious to me when I saw this at first time.